webapp/src/containers/Rhs/projectDetails/index.tsx

```diff
@@ -147,15 +147,17 @@ const ProjectDetails = (projectDetails: ProjectDetails) => {
                                 onClick={handleSubscriptionModal}
                                 className='plugin-btn no-data__btn btn btn-primary project-list-btn'
                             >
-                                {'Create new subscription'}
+                                {'Add new subscription'}
                             </button>
                         </div>
                     </> :
                     <EmptyState
-                        title='No Subscription Present'
-                        subTitle={{text: 'Create a subscription by clicking the button below'}}
-                        buttonText='Create new subscription'
+                        title='No subscriptions yet'
+                        subTitle={{text: 'You can link a subscription by clicking the below button.'}}
```

Comment:
```suggestion
                        subTitle={{text: 'You can create a subscription by clicking the below button.'}}
```

Reply:
maybe?

Reply:
yes, sounds good, however `link` is mentioned in Figma

Reply:
`add` sounds more appropriate as per Figma